package.json

```diff
@@ -88,6 +88,7 @@
     "shortid": "^2.2.16",
     "uuid": "^9.0.0",
     "validator": "^13.7.0",
+    "vite": "^3.2.7",
```

Comment:
Why do we need `vite`?

Reply:
vite is in the package.json on the main branch too, so that's why it's present here.

Reply:
Remove it as we will be merging `develop` into `main` soon.